src/protocols/ethereum/views/TransferSendModal.vue

```diff
@@ -0,0 +1,133 @@
+<!-- eslint-disable vue/no-v-model-argument -->
+<template>
+  <TransferSendBase
+    v-bind="$attrs"
```

Comment:
I strongly suggest to avoid passing all attrs to child components. During my 1,5 year of work here I know that this approach was causing so many issues :(
It's unpredictable and hard to maintain.